### PR TITLE
Bug 1217485 - Update mtbf test to reflect new API for cards_view

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

File: mtbf_driver/tests/mtbf/cards_view/test_mtbf_cards_view_kill_apps_with_two_apps.py [hunk 1]
```diff
@@ -30,11 +30,11 @@ def test_kill_app_from_cards_view(self):
         self.cards_view.wait_for_cards_view()
 
         # Wait for first app ready
-        self.cards_view.wait_for_card_ready(self._test_apps[1])
+        cards_view.cards[1].wait_for_centered()
```
Comment (zapion) [inline mtbf_driver/tests/mtbf/cards_view/test_mtbf_cards_view_kill_apps_with_two_apps.py]: does it need a prefix self. ?

**Comment by zapion** [comment]
Thanks for the patch @mwargers !

File: mtbf_driver/tests/mtbf/cards_view/test_mtbf_cards_view_kill_apps_with_two_apps.py (hunk 1 above)
Comment (mwargers) [inline mtbf_driver/tests/mtbf/cards_view/test_mtbf_cards_view_kill_apps_with_two_apps.py]: Oops, thanks, updated.

**Comment by zapion** [comment]
lgtm.

**Pull request merged**